Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs

```diff
@@ -150,10 +156,10 @@ protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
 				((IListViewController)e.NewElement).ScrollToRequested += OnScrollToRequested;
 
 				nativeListView.DividerHeight = 0;
+				nativeListView.Adapter = _adapter = e.NewElement.IsGroupingEnabled && e.NewElement.OnThisPlatform().IsFastScrollEnabled() ? new GroupedListViewAdapter(Context, nativeListView, e.NewElement) : new ListViewAdapter(Context, nativeListView, e.NewElement);
 				nativeListView.Focusable = false;
```

Comment:
This line triggers Android to want to resurrect the C# version of the adapter. By moving the assignment of the new adapter before this line, Android no longer asks to resurrect the old adapter -- presumably because it finds the one we just assigned.